snuba/subscriptions/executor_consumer.py

```diff
@@ -186,15 +193,18 @@ def create_with_partitions(
         commit: Callable[[Mapping[Partition, Position]], None],
         partitions: Mapping[Partition, int],
     ) -> ProcessingStrategy[KafkaPayload]:
+
         return ExecuteQuery(
             self.__dataset,
             self.__entity_names,
             self.__max_concurrent_queries,
             self.__total_concurrent_queries,
+            self.__partition_count,
             self.__stale_threshold_seconds,
             self.__metrics,
             ProduceResult(self.__producer, self.__result_topic, commit),
             commit,
+            partitions,
```

Comment:
partition_count and partitions are only being used to calculate the total_concurrent queries. Yet we are passing `total_concurrent_queries`, `partition_count` AND `partitions` through. Shouldn't we just be replacing the `total_concurrent_queries` and leaving this argument list unchanged?

Reply:
these are not calculating `total_concurrent_queries` they are calculating `max_concurrent_queries`. We could pass in something like `calculated_max_concurrent_queries` or something like that instead if that's what you mean